website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

```diff
@@ -17,6 +17,12 @@ To set up a job to generate docs:
 3. Under "Execution Settings," select **Generate docs on run**.
    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/documentation-job-execution-settings.png" title="Setting up a job to generate documentation"/>
 
+:::tip
```

Comment:
I think this needs a title. I was able to get this working with the block below

```
:::tip Use dbt compile

To create and schedule a documentation-only job at the end of your production jobs, add the `dbt compile` command under **Commands**. 

:::
```

Reply:
thank you again! i've tweaked it a bit so it flows after the steps and changed the title so it's more catered to those that want to run a docs-only job